jbcli/jbcli/cli/jb.py

```diff
@@ -546,8 +472,6 @@ def get_environment_interactively(env):
 def stop(ctx, clean, env):
     """Stop a running juicebox in this environment
     """
-    env = get_environment_interactively(env)
-    os.chdir("./environments/{}".format(env))
```

Comment:
why were these lines deleted? it means that `jb stop` now no longer honors its `--env` argument

Reply:
I guess maybe what we want is to remove the `--env` argument and just change this to `os.chdir(DEVLANDIA_DIR)`?

Reply:
that makes sense, we don't really need the env exactly because there's no envs anymore and the concept of env is really just an image name used in start and we arent trying to support multiple running instances of juicebox